doc/plotting.rst

```diff
@@ -197,6 +197,16 @@ It is required to explicitly specify either
 Thus, we could have made the previous plot by specifying ``hue='lat'`` instead of ``x='time'``.
 If required, the automatic legend can be turned off using ``add_legend=False``.
 
+Data along x-axis
```

Comment:
"Rotated line plots" as title? Just a suggestion

Reply:
Rotation makes me think of rotating the lines by an angle. How about "Coordinate along y-axis"?